libs/androidlib/src/mill/androidlib/AndroidAppModule.scala

```diff
@@ -800,6 +800,28 @@ trait AndroidAppModule extends AndroidModule { outer =>
 
   }
 
+  def knownProguardRules: T[String] = Task {
+    // TODO need also collect rules from other modules,
```

Comment:
this TODO should change (because Android lib modules exist) - to pick proguard files from [[moduleDeps]] too